chainer/training/extensions/evaluator.py

```diff
@@ -195,11 +195,10 @@ def evaluate(self):
         return summary.compute_mean()
 
     def finalize(self):
-        """Finalizes the updater object.
+        """Finalizes the iterator objects.
```

Comment:
Hm, as this is a finalization method of evaluator, how about saying `Finalizes the evaluator object.`?
User-defined class that inherits `Evaluator` may want to do other finalization things for evaluator by overriding this method.  In that case the docs looks odd.
This rule fits with the fact that pydoc of Updater's `finalize` methods says `Finalizes the updater object.`

Reply:
mmm... looks like I've been mad today.